index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Karla:wght@400&amp;display=swap"
+    />
+    <link rel="stylesheet" href="main.css" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Sample page</title>
+  </head>
+  <body>
+    <main>
```

Comment:
There should not be any style on main, make a div named inside it and do styling on that, you might need another child div  container to adjust height of it